concepts/Common/TypeScript Guides/15 Events/05 Event Objects.md

```diff
@@ -0,0 +1,19 @@
+Each component has event types that define arguments for handling events. Naming convention is the following: the argument type of the `contentReady` event handler is `ContentReadyEvent`.
+
+Events are unions built up from [common and specific parts](/Documentation/Guide/Common/TypeScript_Guides/Events/#Writable_Fields) (to simplify reading them):
```

Comment:
I'm not sure I'm 100% correct and let me know if I'm not. But the phrase "common and specific parts" is too vague. 

```suggestion
Events are unions that build [specific implementation on top of basic event types](/Documentation/Guide/Common/TypeScript_Guides/Events/#Writable_Fields). This implementation makes event definitions easily readable:
```